lib/plugins/alexa-plugin.md

```diff
@@ -1,100 +1,274 @@
 ======================================
 
-> Please note that your hosted montitor must support the Alexa plug-in which exposes an endpoint to the Alexa skill you will create below. Please see [updating my version here](https://github.com/nightscout/cgm-remote-monitor#updating-my-version) for steps
+## Overview
 
-##Setup
+To add Alexa support for your Nightscout site, here's what you need to do:
 
-### 1) Make sure to add `alexa` to the list of plugins in your `ENABLE` setting (note: environment variables are set in the configuration section for your monitor typically Azure, Heroku, etc.)
-### 2) Sign in to https://developer.amazon.com/ and navigate to the "Alexa" tab. Select "Getting started" in "Alexa Skills Kit" NOTE: You must sign in to the developer account using the same Amazon account as you use to manage your Echo device.
-    * Click on "Add a new skill". Fill in "Nightscout" as the name and "nightscout" as the invocation name (feel free to use other names as you see fit).
-    * This skill will not use the "Audio Player".
-    * Click Next
-### 3) Enter the following in the "Intent schema" box
+1. [Activate the `alexa` plugin](#activate-the-nightscout-alexa-plugin) on your Nightscout site, so your site will respond correctly to Alexa's requests.
+1. [Create a custom Alexa skill](#create-your-alexa-skill) that points at your site and defines certain questions you want to be able to ask. (You'll copy and paste a basic template for this, to keep things simple.)
 
-```javascript
+To add Alexa support for a plugin, [check this out](#adding-alexa-support-to-a-plugin).
+
+## Activate the Nightscout Alexa Plugin 
+
+1. Your Nightscout site needs to be new enough that it supports the `alexa` plugin. It needs to be [version 0.9.1 (Grilled Cheese)](https://github.com/nightscout/cgm-remote-monitor/releases/tag/0.9.1) or later. See [updating my version](https://github.com/nightscout/cgm-remote-monitor#updating-my-version) if you need a newer version.
+1. Add `alexa` to the list of plugins in your `ENABLE` setting. ([Environment variables](https://github.com/nightscout/cgm-remote-monitor#environment) are set in the configuration section for your monitor. Typically Azure, Heroku, etc.)
+
+## Create Your Alexa Skill
+
+### Get an Amazon Developer account
+
+- Sign up for a free [Amazon Developer account](https://developer.amazon.com/) if you don't already have one.
+- [Register](https://developer.amazon.com/docs/devconsole/test-your-skill.html#h2_register) your Alexa-enabled device with your Developer account.
+- Sign in and go to the [Alexa developer portal](https://developer.amazon.com/alexa). 
+
+### Create a new Alexa skill
+
+1. Select "Alexa Skills Kit" in the main menu bar.
+1. Click the "Start a Skill" button. This will take you to the Skills console.
+1. Click the "Create Skill" button on the Skills console page.
+1. Name your new skill "Nightscout" (or something else, if you like). Use English (US) as your language. Click "Next".
```

Comment:
I'm not really happy about tellling to use language english without further note that the own language can be used if the corresponding translations are available...